src/v2/view-builder/views/email/BaseAuthenticatorEmailView.js

```diff
@@ -77,6 +75,7 @@ const Body = BaseFormWithPolling.extend(Object.assign(
 
     triggerAfterError(model, error) {
       BaseForm.prototype.triggerAfterError.apply(this, arguments);
+      const isFormPolling = this.polling;
```

Comment:
wondering we `stopPolling` after and assigned variable before?

Reply:
It is to be on the safer side so we won't have multiple polling going without a way to cancel.

Reply:
nit: `const isFormPolling = !!this.polling;`